### PR TITLE
Cleanup js

### PR DESCRIPTION
**Pull request by camrossi**

### CODE REVIEW

**Comment by cpaggen** [comment]
I am not approving it. We must stop introducing changes we did not discuss such as upgrading the python distro. We need to isolate why we are seeing browsers hang. It was not a problem with v1.0, so some sort of change caused the new behavior.

File: terraform/TEMPLATES/calico_nodes.html [hunk 1]
```diff
@@ -142,14 +142,7 @@ <h2>K8s Cluster Topology</h2>
...
    </div>
 <script src="{{ url_for('static',filename='assets/cui-2.0.0/js/jquery-3.0.0.min.js') }}"></script>
-<script src="{{ url_for('static',filename='assets/cui-2.0.0/js/lodash.min.js') }}"></script>
-<script src="{{ url_for('static',filename='assets/cui-2.0.0/js/clipboard.js') }}"></script>
-<script src="{{ url_for('static',filename='assets/cui-2.0.0/js/styleguide.js') }}"></script>
-<script src="{{ url_for('static',filename='assets/cui-2.0.0/js/highcharts.js') }}"></script>
-<script src="{{ url_for('static',filename='assets/cui-2.0.0/js/exporting.js') }}"></script>
-<script src="{{ url_for('static',filename='assets/cui-2.0.0/js/html2canvas.js') }}"></script>
 <script src="{{ url_for('static',filename='assets/cui-2.0.0/js/functionLibrary.js') }}"></script>
-<script src="{{ url_for('static',filename='assets/js/utils.js') }}"></script>
```
Comment (samiib) [inline terraform/TEMPLATES/calico_nodes.html:152]: Like Shan mentioned the utils.js is actually used. So we shouldn't be removing this one.

File: terraform/static/assets/cui-2.0.0/css/frame.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
... !important}.tree-box .child-tree .child-tree{padding-left:20px !important}.tree-box ul{list-style:none;padding:0;display:flex;flex-direction:row;border-bottom:1px solid #f2f2f2;padding:8px 10px 8px 0px}.tree-box ul li span.icon-add-outline,.tree-box ul li span.icon-dc-power{cursor:pointer}.tree-box ul li:first-child{width:20px;position:relative;top:0px;flex:none}.tree-box ul li:nth-child(2){flex:1}.tree-box ul li{flex:1}.tree-box ul li span.active{color:#6ebe4a}body.cui .header{position:relative !important}div#cubex{min-height:100%;position:relative;height:100%}.cui .table--bordered td{border-top:1px solid #dfdfdf;border-right:1px solid #dfdfdf;font-size:14px}.p10{padding:10px 0 !important}.mr5{margin-right:5px !important}.bg-gray{background-color:#f5f5f6}.bg-White{background-color:#fff}.shadow-right{box-shadow:2px 2px 4px #d9d2d2}.shadow-all{box-shadow:0px 1px 4px #b4b4b4}.flexrow{display:flex;justify-content:flex-start;align-items:stretch;flex-wrap:nowrap;padding:10px}.cell{min-height:75px;flex-basis:100%}.cui .form-group{margin:10px 0 0}body.cui{overflow:auto !important}.cui ul.tabs>li>a{padding:15px 0 !important}.cui ul.tabs>li>a p{padding:0px !important;margin:0px !important}
```
Comment (samiib) [inline terraform/static/assets/cui-2.0.0/css/frame.css:1]: What happened to the frame.css? Why did it change so much?

File: terraform/static/assets/cui-2.0.0/js/highcharts.js [hunk 3]
```diff
@@ -488,5 +488,4 @@ this.options.events.mouseOver&&r(this,"mouseOver");this.setState("hover");a.hove
...
 u={};e(c,this.options,u,0);return u}});M(J,"masters/highcharts.src.js",[J["parts/Globals.js"],J["parts/Utilities.js"]],function(c,e){var F=e.extend;F(c,{animObject:e.animObject,arrayMax:e.arrayMax,arrayMin:e.arrayMin,attr:e.attr,correctFloat:e.correctFloat,defined:e.defined,destroyObjectProperties:e.destroyObjectProperties,discardElement:e.discardElement,erase:e.erase,extend:e.extend,extendClass:e.extendClass,isArray:e.isArray,isClass:e.isClass,isDOMElement:e.isDOMElement,isNumber:e.isNumber,isObject:e.isObject,
-isString:e.isString,numberFormat:e.numberFormat,objectEach:e.objectEach,offset:e.offset,pad:e.pad,pick:e.pick,pInt:e.pInt,relativeLength:e.relativeLength,setAnimation:e.setAnimation,splat:e.splat,syncTimeout:e.syncTimeout,wrap:e.wrap});return c});J["masters/highcharts.src.js"]._modules=J;return J["masters/highcharts.src.js"]});
-//# sourceMappingURL=highcharts.js.map
```
Comment (samiib) [inline terraform/static/assets/cui-2.0.0/js/highcharts.js:492]: If you're not planning to use the js files, may as well delete them.

File: .github/workflows/pipeline.yaml [hunk 4]
```diff
@@ -20,7 +20,7 @@ jobs:
         - name: Install Python 3
           uses: actions/setup-python@v1
           with:
-            python-version: 3.10.2
+            python-version: 3.10.4
```
Comment (samiib) [inline .github/workflows/pipeline.yaml:23]: Anything to change with python version for unit testing has a really really low risk of changing anything in NKT.
So this change shouldn't be an issue.

**Review by samiib — COMMENTED** [review]
Just a few things I have mentioned within the review.

**Comment by camrossi** [comment]
> Why go from Python 3.10.2 to 3.10.4 - this is not a change we planned, it risks introducing instability.

As discussed over chat this was only done for the unit test and only because 3.10.2 is no more available.

**Pull request merged**